examples/with-cookie-auth-fauna/pages/profile.js

```diff
@@ -1,17 +1,19 @@
-import cookie from 'cookie'
-import Router from 'next/router'
+import useSWR from 'swr'
 import { withAuthSync } from '../utils/auth'
-import { FAUNA_SECRET_COOKIE } from '../utils/fauna-auth'
-import { profileApi } from './api/profile'
 import Layout from '../components/layout'
 
-const Profile = (props) => {
-  const { userId } = props
+const fetcher = async () => {
+  let response = await fetch('/api/profile')
+  let data = await response.json()
+  return data
+}
 
+const Profile = () => {
+  const { data, error } = useSWR('profile', fetcher)
+  if (error) return <p>{error.message}</p>
```

Comment:
@todortotev Redirect the user using to `/login` using `router.push('/login')` if an user wasn't found.